.github/workflows/run_tests.yml

```diff
@@ -10,38 +10,78 @@ env:
   POETRY_VERSION: "1.6.1"
 
 jobs:
+  lint:
+    name: Lint
```

Comment:
Is this linting or because of --fix-only fixing? If it's fix, we should name it fix.

Reply:
Ideally, it would Lint + auto fix and error if there are any lint errors, but right now there are 1600 ruff errors that arent auto-fixable. 

Maybe can add a check to only error on modified files, so the community will gradually refactor the modules in order to get their PR merged?

Reply:
Ideally yeah, let's call it 'Fix' for now and then circle back on the error check on modified files